### ISSUE TITLE
You're Outside your Defined Radius

### ISSUE THREAD

**Issue by Kriannor** [NECROBOTIO/NecroBot#3796]
Been having this issue since 0.7.8 yesterday.

Double checked my config and everything is as it should, my Latitude and Longitude are where they are supposed to be. Tried changing my LastPos.ini to reflect my default position, tried deleting it also. Tried changing my default position. Tried using a fresh out of setup config.json. Nothing works.

As soon as I login my initial L&L are incorrect for some reason. Something is changing it away farther than it should, by several thousand KM actually.

Anyone know what could be causing this issue and how to solve it?

**Comment by necr-0** [comment]
Try downloading a clean version of 0.8.0 and try again, making any relevant edits to the coordinates.

**Comment by Kriannor** [comment]
I was using a clean 0.8.0 install. Now I changed my default position to the other side of town and suddenly it works again. I've been trying to position myself around the same area since yesterday. Are there areas where the position simply bugs out? I DID change the position a couple hundred meters and still had the problem, didn't thought going farther would solve it, because that doesn't make much sense in my head.

Can someone confirm there ARE places the Necro simply won't work? I'm sorry if that's a known issue, been using it for just a couple of days now.

Also, is there a way to turn back on that "[INFO] Walking to Pokestop XXXm away (xx seconds)"? I enjoyed that on 0.7.6.
